control/controlclient/direct.go

```diff
@@ -1582,8 +1582,8 @@ func postPingResult(start time.Time, logf logger.Logf, c *http.Client, pr *tailc
 
 // ReportHealthChange reports to the control plane a change to this node's
 // health.
```

Comment:
Document which args can be nil and what that means.

Looks like `w` must be non-nil and `us` can be nil which means healthy?

Reply:
done